### DISCUSSION TITLE
Documentation - User manual

### DISCUSSION THREAD

**Discussion by mturra-Pro** [glowroot/glowroot#839] [General]
I found all documentation about agent and configuration, but about the features.

Today, for instance, in Transaction -> Slow Traces a blue dot appear. 
What the blue color means?

Thanks

**Comment by trask** [comment]
blue color means the request is still active, probably this note should be added to the tooltip on that page (or maybe a legend)

**Reply by mturra-Pro** [reply]
I deduced it from the source code. But it would be nice to have the legend/tooltip and the user manual too.
